src/site/_includes/js/eligibility.js

```diff
@@ -2574,8 +2583,6 @@ function vaPensionResult(input) {
 
   const meetsDischargeReq = not(isOneOf(input.dischargeStatus, [
     'dishonorable',
-    'oth',
```

Comment:
The eligibility condition text should be updated to match this new logic.  "Have a discharge status that is not dishonorable" (like for VA disability)